lib/zaputil/stack_extact_core.go

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2018 Yandex LLC. All rights reserved.
```

Comment:
typo in filename: stack_extact_core instead of stack_extRact_core

Reply:
fixed